samples/apps/org.wso2.carbon.uuf.sample.pets-store.app/src/main/pages/pets/{id}.js

```diff
@@ -1,8 +1,24 @@
+/*
+ *  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This file is not a new file, I think it was created in last year. That is why I added 2016. So should this be changed to 2017?